src/pc/configfile.c

```diff
@@ -99,6 +100,7 @@ static const struct ConfigOption options[] = {
     {.name = "vsync",                .type = CONFIG_TYPE_UINT, .uintValue = &configWindow.vsync},
     {.name = "texture_filtering",    .type = CONFIG_TYPE_UINT, .uintValue = &configFiltering},
     {.name = "master_volume",        .type = CONFIG_TYPE_UINT, .uintValue = &configMasterVolume},
+    {.name = "music_enabled",        .type = CONFIG_TYPE_UINT, .uintValue = &configMasterVolume},
```

Comment:
I think you meant `configMusicVolume` here. Also might be a good idea to rename it to `music_volume`.

Reply:
Fixed on the last commit. Thanks